### ISSUE TITLE
Draft rules for tweet contents

### ISSUE THREAD

**Issue by IstoraMandiri** [ethereumclassic/tweets-etc_network#15]
There should be limits on what kind of tweets are published

- No spam
- No scams
- No drama
- Stay classy

Anything else?

**Comment by w1g0** [comment]
No drama and stay classy is to vague. It's better to specify imho. How are you gonna check if it ain't a scam? Again 4 bullet points is a good start for a sense of direction.
Stating what you can't do gives it a negative impression. Setting expectations seems like a more positive approach.

**Comment by Soteria-Smart-Contracts** [comment]
- Tweets cannot be directly or indirectly looking for people to buy a service or an asset 
- If it is a retweet, the same rule applies.
- Tweets about projects must be straightforward in their reasoning, so no things like (Follow *account* to stay updated!)
- If a tweet is about a project, it must be marked at the end if that tweet was pushed by the owner or contributor to that project

**Comment by TheCrowbill** [comment]
- All tweets must conform to the Twitter rules and policies found [here.](https://help.twitter.com/en/rules-and-policies)

**Comment by IstoraMandiri** [comment]
Only link to apps/projects via the eth.org website, so that people can see the disclaimer.

**Comment by IstoraMandiri** [comment]
> Only link to apps/projects via the eth.org website, so that people can see the disclaimer.

Another approach to this is https://github.com/ethereumclassic/twitter-together/pull/40

Basically, we can retweet other twitter handles as long as they only contain a link to the project on our website. I think this is a good approach as it naturally creates cross-promotion

**Comment by IstoraMandiri** [comment]
Until we have a better way of doing this, I encourage all maintainers to copy and paste the verified line in the preview before making the first approval similar to https://github.com/ethereumclassic/twitter-together/pull/41#issuecomment-1246620402

The preview can be checked in the `checks` tab, and has to be manually triggered for new contributors

**Comment by brolal** [comment]
this may help for a starting point: 
Accuracy. in photographs and design and sound and video clips as well as in articles. 
Independence. fair, transparent, professional, and helpful.
Disclose. relevant information that may have influenced or affected $ETC. decisions.
Provide readers with sufficient information
Meaningful context. Assess and judge the material we publish.

**Comment by w1g0** [comment]
> @w1g0 You mentioned this regarding retweet etiquette, how opinionated is too opinionated?
> 
> For context, the retweet of https://twitter.com/ChuckSRQ/status/1572353721228742656 may be considered too opinionated.
> 
> > I gave up the Ethereum Classic twitter account because I did not want to devote the time and work required. If the team running it now, doesn’t believe in the project, they should give it to someone that does want to put in the work.
> 
> Would "Retweet != Endorsement" disclaimer in bio would help?
> 
> [90ebec6#commitcomment-84623711](https://github.com/ethereumclassic/twitter-together/commit/90ebec66fd037857756328cdd59bdb857a46f47e#commitcomment-84623711)

If the content has value to contribute without invoking the possibility of further drama, the solution I suggest is to rewrite the content and not to retweet it.

No disclaimer, keep the messaging positive, relevant, objective and no negative Nancy's on the steering wheel of the ETC starship.

**Comment by w1g0** [comment]
With regards to the EIP (https://github.com/ethereumclassic/twitter-together/pull/65) tweet suggested by @bobsummerwill, regardless if I support the idea or not, it gives the impression that when proposer and reviewer agree it reflects the support of the whole ETC community which was further explained in the reasoning. If community members support a certain idea they are able to retweet with their account and not use the @etc_network handle to amplify their ideas. Please, I encourage to be more objective with regards to what has been decided and ask unbiased questions to hear the voice of the community.

**Comment by IstoraMandiri** [comment]
I think anyone should be able to propose tweets, even (or especially) if it is their own idea. 

As proposers cannot approve their own proposals, it is a separate issue of whether or not such proposals get approved, and should be done so objectively as possible using a well defined and agreed-upon checklist, as we are attempting to define here.

Once this checklist is defined, we should require 2 approvals, but right now it's set to 1 to facilitate testing.

**Comment by IstoraMandiri** [comment]
For the record, I am intentionally leaning towards approving tweets that are not obviously objectionable or self-promotional, as this encourages contributions.

**Comment by IstoraMandiri** [comment]
@w1g0 we should absolutely not interpret retweets as endorsements and make this clear to followers. 

Otherwise, we'd be unable to retweet anything, and we cannot reflect the diverse set of views of the ETC community.

For exmaple, during the SHA3 debate, as was with the community calls, it was better to amplify both sides of the debate, rather than attempting to avoid it all together.

**Comment by IstoraMandiri** [comment]
I think that ideally we should be tweeting to multiple accounts with their own set of rules, inheriting from an overall set of basic rules, possible after #66

**Comment by w1g0** [comment]
>I think anyone should be able to propose tweets, even (or especially) if it is their own idea.

Sure, anyone can suggest whatever they want but are you going to review all the proposals? Even if someone automates a Denial Of Service attack? This will get messy real soon, especially in bull markets.

>@w1g0 we should absolutely not interpret retweets as endorsements and make this clear to followers.

There's a way you want it to operate (intention) and how the majority on Twitter interprets retweets (repercussion), this is prone to evil based on good intention. A simple disclaimer won't change this...

ps: For now I will do my best to only review factual statements and question subjective ones until better rules are written out. You made clear that this is still in Beta, that might work in your head @IstoraMandiri but the majority is not aware. Here a disclaimer as signature might work imho.

**Comment by IstoraMandiri** [comment]
Donald mentioned on the latest call that talking about price will generate interest. 

Discussed on https://youtu.be/kOJOb-iA9uI?t=3159

**Comment by IstoraMandiri** [comment]
Case studies:

https://twitter.com/bitcoin
https://twitter.com/ethereum

BTC + ETH = ETC ?

**Comment by IstoraMandiri** [comment]
https://github.com/ethereumclassic/tweets-eth_classic/pull/15

We should always get a confirmation from the original submitter before merging.

**Comment by bobsummerwill** [comment]
Moved to https://github.com/ethereumclassic/tweets-eth_classic/issues/16.

**Comment by w1g0** [comment]
> Moved to [ethereumclassic/tweets-eth_classic#16](https://github.com/ethereumclassic/tweets-eth_classic/issues/16).

@bobsummerwill, I would leave this issue open for inspiration and to define the rules for @etc_network as it might slightly differ to serve a complimentary purpose.
